ui/index.html

```diff
@@ -10,6 +10,15 @@
 
   <body>
     <script src="app.js"></script>
-    <script>Elm.Main.fullscreen()</script>
+    <script>
+    const hash = location.hash;
+    location.hash = "";
+    const app = Elm.Main.fullscreen();
+    app.ports.onFetched.subscribe(() => {
+      if (hash) {
+        setTimeout(() => location.hash = hash, 100);
```

Comment:
These 100ms is the amount of time required for the browser to fully render the list of releases after they've been received. We can't precisely know this value, so I fiddled a little and came with `100`.